src/main/webapp/WEB-INF/tags/ksr/securityRequestRoleQualifications.tag

```diff
@@ -0,0 +1,85 @@
+<%@ include file="/jsp/sys/kfsTldHeader.jsp" %>
+
+<%@ attribute name="securityRequestRoleIndex" required="true"
+              description="Index of the security request role instance in the document collection to render qualifications for." %>
+
+<c:set var="securityRequestRole" value="${KualiForm.document.securityRequestRoles[securityRequestRoleIndex]}" />
+
+<c:set var="securityRequestRoleQualificationHdr" value="${securityRequestRole.newRequestRoleQualification}" />
+<c:if test="${readOnly}">
+  <c:set var="securityRequestRoleQualificationHdr" value="${securityRequestRole.requestRoleQualifications[0]}" />
+</c:if>
+
+<table border="0" cellpadding="0" cellspacing="0" class="celltable">
+    <%-- Header Row --%>
+    <tr>
+        <th width="10" scope="col">&nbsp;</th>
+        <c:forEach var="qualificationDetail" items="${securityRequestRoleQualificationHdr.roleQualificationDetails}">
+          <th scope="col">
+             <c:set var="attributeEntry" value="${qualificationDetail.attributeEntry}" />
+             <kul:htmlAttributeLabel attributeEntry="${attributeEntry}" useShortLabel="false" noColon="true"/>
+          </th>
+        </c:forEach>
+
+        <c:if test="${!readOnly}">
+            <th scope="col">Actions</th>
+        </c:if>
+    </tr>
+
+    <%-- Add Line --%>
+    <c:if test="${!readOnly}">
+        <tr>
+            <th scope="row"><div align="right">add:</div>
+            </th>
+            
+            <c:forEach var="qualificationDetail" items="${securityRequestRoleQualificationHdr.roleQualificationDetails}" varStatus="status">
+              <td>
+                <c:set var="attributeEntry" value="${qualificationDetail.attributeEntry}" />
+                <kul:htmlControlAttribute attributeEntry="${attributeEntry}"
+                    property="document.securityRequestRoles[${securityRequestRoleIndex}].newRequestRoleQualification.roleQualificationDetails[${status.count-1}].attributeValue" readOnly="${readOnly}" />
+                
+                <c:set var="attributeDefinition" value="${qualificationDetail.attributeDefinition}" />    
+                <c:if test="${(!empty attributeDefinition.quickFinder) && !readOnly}"> 
+                	<c:set var="quickFinderAttributeDefinition" value="${qualificationDetail.attributeDefinition.quickFinder}" />

```

Comment:
If this is just a QuickFinder, then the variable should be renamed for simplicity, especially if the "securityRequestQualifierLookup" tag changes the name on its related attribute. The sames goes for this variable setup in the existing-lines section.

Reply:
updated name